content-scripts/prototype-handler.js

```diff
@@ -23,7 +23,9 @@ function injectPrototype() {
   };
 }
 
-if (!(document.documentElement instanceof SVGElement) && location.pathname.split("/")[1] === "projects") {
+const isLocal =
+  location.origin === "https://scratchfoundation.github.io" || location.origin.startsWith("http://localhost:");
+if ((!(document.documentElement instanceof SVGElement) && location.pathname.split("/")[1] === "projects") || isLocal) {
```

Comment:
Locally run WWW instances should not generally trigger this.

Reply:
What do you think this should be changed to?
Checking location.port (which should match GUI port) instead of the origin?

Reply:
Yeah, the code looks good now